.github/workflows/dhis2-verify-app.yml

```diff
@@ -0,0 +1,174 @@
+              with:
+                  node-version: 18.x
+
+            - name: Install
+              run: yarn install --frozen-lockfile
+
+            - name: Build
+              run: yarn d2-app-scripts build
+
+            - uses: actions/upload-artifact@v3
+              with:
+                  name: app-build
+                  path: |
+                      **/build
+                      !**/node_modules
+                  retention-days: 1
+
+    lint:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: 18.x
+
+            - name: Install
+              run: yarn install --frozen-lockfile
+
+            - name: Generate translations
+              run: yarn d2-app-scripts i18n generate
+
+            - name: Lint
+              run: yarn d2-style check
+
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: 18.x
+
+            - name: Install
+              run: yarn install --frozen-lockfile
+
+            - name: Generate translations
+              run: yarn d2-app-scripts i18n generate
+
+            - name: Test
+              run: yarn d2-app-scripts test
+
+    e2e-prod:
+        runs-on: ubuntu-latest
+        needs: test
+        if: "!contains(github.event.head_commit.message, '[skip ci]')"
+
+        strategy:
+            matrix:
+                containers: [1, 2, 3, 4]
+
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: 18.x
+
+            - uses: actions/cache@v2
+              id: yarn-cache
+              with:
+                  path: '**/node_modules'
+                  key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+            - name: Install Cypress binary
+              run: yarn cypress install
+
+            - name: End-to-End tests
+              uses: cypress-io/github-action@v2
+              with:
+                  record: true
+                  parallel: true
+                  start: ${{ env.SERVER_START_CMD }}
+                  wait-on: ${{ env.SERVER_URL }}
+                  wait-on-timeout: 300
+                  cache-key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+                  group: 'e2e'
+                  tag: ${{ github.event_name }}
+              env:
+                  CI: true
+                  CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+                  COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
+                  SERVER_START_CMD: 'yarn cypress:start'
+                  SERVER_URL: 'http://localhost:3000'
+                  cypress_dhis2_api_stub_mode: 'DISABLED'
+                  REACT_APP_DHIS2_BASE_URL: ${{ secrets.CYPRESS_DHIS2_BASE_URL_40 }}
```

Comment:
Is this wanted?
Does it mean we need to update the file whenever 41 is coming out?

Reply:
This is only until we upgrade cypress. I tried reusing the shared workflows but there were cypress version issues. Right now, v40 is the only relevant prod version to test.